src/main/java/com/uber/cadence/internal/replay/WorkflowDecisionContext.java

```diff
@@ -96,11 +95,7 @@ public void accept(Exception cause) {
     final StartChildWorkflowExecutionDecisionAttributes attributes =
         new StartChildWorkflowExecutionDecisionAttributes();
     attributes.setWorkflowType(parameters.getWorkflowType());
-    String workflowId = parameters.getWorkflowId();
-    if (workflowId == null) {
-      workflowId = generateUniqueId();
-    }
-    attributes.setWorkflowId(workflowId);
```

Comment:
Why is it removed? Is not server requiring it anymore?

Reply:
The same reason as above. But this will conflict with your new change and I'll just leave it as is.